diff -r b2f91639f493 pygments/formatters/img.py
--- a/pygments/formatters/img.py	Sun Apr 25 12:45:52 2010 +0200
+++ b/pygments/formatters/img.py	Wed Apr 28 12:43:16 2010 -0400
@@ -18,10 +18,14 @@
 
 # Import this carefully
 try:
-    import Image, ImageDraw, ImageFont
+    from PIL import Image, ImageDraw, ImageFont
     pil_available = True
 except ImportError:
-    pil_available = False
+    try:
+        import Image, ImageDraw, ImageFont
+        pil_available = True
+    except ImportError:
+        pil_available = False
 
 try:
     import _winreg
